src/main.py

```diff
@@ -24,29 +47,31 @@ def readSettingsFromUI():
 
 
 if __name__ == "__main__":
-    camera = cv2.VideoCapture(0)
-    eyeDetector = cv2.CascadeClassifier("resources/haarcascade_eye.xml")
-    uiLaunched = false
-    settings = {}
+    # camera = cv2.VideoCapture(0)
+    # eyeDetector = cv2.CascadeClassifier("resources/haarcascade_eye.xml")
+    # uiLaunched = false
+    # settings = {}
 
-    blinkDuration = 0
-    while True:
-        if uiLaunched:
-            readSettingsFromUI(settings)
+    # blinkDuration = 0
+    # while True:
+    #     if uiLaunched:
+    #         readSettingsFromUI(settings)
 
-        _, image = camera.read()
+    #     _, image = camera.read()
 
-        eyeCoords = detectEyes(image, eyeDetector)
+    #     eyeCoords = detectEyes(image, eyeDetector)
 
-        didBlink = detectBlink(eyeCoords, blinkDuration)
+    #     didBlink = detectBlink(eyeCoords, blinkDuration)
 
-        screenCoords = computeScreenCoords(eyeCoords)
+    #     screenCoords = computeScreenCoords(eyeCoords)
 
-        if didBlink:
-            clickMouse(screenCoords)
+    #     if didBlink:
+    #         clickMouse(screenCoords)
 
-        moveMouse(eyeCoords)
+    #     moveMouse(eyeCoords)
 
-        if menuKeyPressed():
-            launchUIThread()
-            uiLaunched = True
+    #     if menuKeyPressed():
+    #         launchUIThread()
+    #         uiLaunched = True
+    program = IrisSoftware()
+    program.run()
```

Comment:
What is the point of this here? Doing this removes the main loop functionality of our program, some of which was placeholder scaffold (i.e. readSettingsFromUI(), detectBlink()), but most of which was implemented functionality (i.e. camera.read(), detectEyes(), computeScreenCoords(), moveMouse() [in my pending PR]).

Reply:
The other way of doing this might be to pass the frame data from your code to my UI code and then display the camera data that way, which actually sounds a little better. I started doing it this way because it was easier to wrap my head around, but now that I understand it I could migrate it. What's your recommendation?